test/README.md

```diff
@@ -0,0 +1,16 @@
+## Usage
+
+### Prerequisites
+```bash
+pip install pytest
+```
+after changes in pyproject.toml do:
+```bash
+cd .. && pip install -e .
```

Comment:
<img width="519" alt="s" src="https://github.com/Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024/assets/116720366/d0fdca82-a66c-434c-bdc1-dec5b2c3a601">

That's the error I receive when executing this command

Reply:
I would suggest we just use a ";"
```suggestion
cd .. ; pip install -e .
```